src/Analyzer/TableFunctionNode.cpp

```diff
@@ -71,6 +72,13 @@ void TableFunctionNode::dumpTreeImpl(WriteBuffer & buffer, FormatState & format_
         buffer << '\n' << std::string(indent + 2, ' ') << "ARGUMENTS\n";
         arguments.dumpTreeImpl(buffer, format_state, indent + 4);
     }
+
+    if (!settings_changes.empty())
+    {
+        buffer << '\n' << std::string(indent + 6, ' ') << "SETTINGS";
```

Comment:
It seems that we can use indent + 2, here. Because SETTINGS are actually not part of arguments.